<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasCheck</class>
 <widget class="QWidget" name="PasCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="contentWidget" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>567</width>
     <height>532</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: white;
}
#checkPasEdit {
	color: black;
	font-size: 14px;
}
QLabel {
	color: black;
	font-size: 14px;
}
QCheckBox::indicator:unchecked:disabled {
	background-color: #cccccc;
	border-radius: 4px;
}
QCheckBox::indicator:checked:disabled {
	background-color: green;
	border-radius: 4px;
}</string>
   </property>
   <widget class="QLabel" name="pageTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>260</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; font-size: 20px; font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Password Strength Check</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="checkPasEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>64</number>
    </property>
    <property name="placeholderText">
     <string>Enter password to check</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="strengthBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>118</y>
      <width>560</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar::chunk
 {
	border-radius: 5px;
	background-color: red;
}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="timeToBreak">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="bits">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0 bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time needed to break:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bits as encrypted:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Letters:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capital Letters:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numbers:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>140</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Special Characters:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="lowerCheck">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="capitalCheck">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="numberCheck">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="specialCheck">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>77</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="menuWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>500</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>200</width>
     <height>532</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>191</red>
         <green>54</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font-size: 16px;
	font-weight: bold;
	background-color: #bf3636;
	color: white;
	padding: 8px;
	padding-left: 24px;
	border: none;
	border-left: 2px solid white;
	text-align:left;
}
QPushButton:hover {
	background-color: #d43f3f;
}
QWidget{
	background-color: #bf3636;
}</string>
   </property>
   <widget class="QPushButton" name="pasListBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Password List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pasGenBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Password Generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pasCheckBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-left: 2px solid #cccccc;</string>
    </property>
    <property name="text">
     <string>Check Strength</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logOutBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/res/pasman.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
